### PR TITLE
Create base structure 

### PR DESCRIPTION
**Pull request by LHBO19**
#1  

I added the base structure of the application by creating a react-app.

### CODE REVIEW

File: README.old.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# bestiary-web
```
Comment (avelinoschz) [inline README.old.md]: This file isn't needed anymore

Comment (LHBO19) [reply README.old.md]: obligame

Comment (avelinoschz) [inline README.old.md]: This file isn't needed anymore

**Review by avelinoschz — CHANGES_REQUESTED** [review]

File: README.old.md (hunk 1 above)
Comment (LHBO19) [reply README.old.md]: ok

**Review by avelinoschz — APPROVED** [review]

**Review by shibeknight — APPROVED** [review]

**Comment by avelinoschz** [comment]
[React app base structure](https://trello.com/c/WQ0lLoDf/1-react-app-base-structure)

**Pull request merged**